gemfire-core/src/main/java/com/gemstone/gemfire/internal/cache/LocalRegion.java

```diff
@@ -14532,8 +14535,6 @@ private void memTrace(String mesage) {
     }
   }
 
-  public boolean isInternalColumnTable() {
-    return this.getName().toUpperCase().endsWith(StoreCallbacks.SHADOW_TABLE_SUFFIX);
-  }
+  public boolean isInternalColumnTable = false;
```

Comment:
Getter method like before should remain that returns the value and this should be made private (no one should be able to change the value from outside).